test/Microsoft.AspNetCore.Mvc.FunctionalTests/HtmlHelperOptionsTest.cs

```diff
@@ -48,12 +50,11 @@ public HtmlHelperOptionsTest(MvcTestFixture<RazorWebSite.Startup> fixture)
         }
 
         [Fact]
-        [ReplaceCulture]
```

Comment:
Don't remove. The error message comes from our resources and may vary (eventually).